pycarol/bigquery.py

```diff
@@ -254,6 +255,31 @@ def _generate_client(service_account: T.Dict) -> bigquery.Client:
         client = bigquery.Client(project=project, credentials=credentials)
         return client
 
+    def _build_query_job_labels(self) -> T.Dict[str, str]:
+        label_values = [
+            self._env.get("env_id", ""),
+            self._env.get("env_name", ""),
+            self._env.get("org_id", ""),
+            self._env.get("org_name", ""),
+            "sync",
+            "py_carol",
+            "",
+            self._env.get("app_name", ""),
+            "",
+        ]
+        label_keys = [
+            "tenant_id",
+            "tenant_name",
+            "organization_id",
+            "organization_name",
+            "job_type",  ## Is any case that using this class is async?
+            "source",
+            "task_id",  ## In this case may we use the "mdmBigQueryProvisionId" from 'carol._current_env()'?
+            "carol_app_name",
+            "carol_app_process_name",  ## I do not understand what should be this property
+        ]
+        return {k: v for k, v in zip(label_keys, label_values) if v.strip() != ""}
```

Comment:
why not creating the dict directly instead of creating keys and values and zipping them?

Reply:
Do you mean like this?
```python
labels = {
  "tenant_name" : "tenant_name",
  "carol_app_name" : "",
  ...
}
return {k:v for k,v in labels.items() if v.strip() != ""}
``` 
Sure, makes sense.

Reply:
yep.